src/main/java/org/springframework/social/bitbucket/api/BitBucketUser.java

```diff
@@ -41,7 +41,7 @@
     private String lastName;
 
     @JsonProperty("is_team") @Getter
-    private boolean team;
+    private Boolean isTeam;
```

Comment:
Is there really a reason to use a `Boolean` here ? If there is, a comment indicating in which case it happens would be nice. And a `@Nullable` annotation would probably make sense as well.